src/model/migrations.ts

```diff
@@ -680,6 +680,19 @@ export default async function runMigrations() {
 
   migrations.push(v17);
 
+  /*
+   *************** Migration v18 ******************
+   * Removes old MMKV cache
```

Comment:
```suggestion
   * Removes old MMKV cache for Imgix images
```

I'd add a mention that this only touches image part.